packages/compass-crud/src/utils/cancellable-queries.spec.ts

```diff
@@ -40,7 +40,7 @@ describe('cancellable-queries', function () {
     dataService = await connect(info.connectionOptions);
 
     currentOpsByNS = async function (ns) {
-      const ops = await dataService.currentOp(false);
```

Comment:
Only the currentOp test was using true value, so we can drop the argument and always use `allUsers: true` in the `$currentOp` stage.